test/jdk/java/rmi/registry/nonLocalRegistry/NonLocalRegistryTest.java

```diff
@@ -46,22 +48,29 @@
  * Login or ssh to the different host and invoke {@code $JDK_HOME/bin/rmiregistry}.
  * It will not show any output.
  *
- * On the first host modify the @run command above to replace "rmi-registry-host"
- * with the hostname or IP address of the different host and run the test with jtreg.
+ * On the first host, run the test and provide the hostname or IP address of the
+ * different host when prompted in the console, or run jtreg command with system
+ * property -Dregistry.host set to the hostname or IP address of the different host.
  */
-public class NonLocalRegistryTest {
+public class NonLocalRegistryTest extends NonLocalRegistryBase {
```

Comment:
Move  private static void assertIsAccessException(Throwable ex) method to NonLocalRegistryBase class.